PostBuild.sh

```diff
@@ -118,6 +119,23 @@ function CreateFstab {
         err_exit "Failed setting up /etc/fstab"
    fi
 
+   # Add any swaps to fstab
+   mapfile -t SWAP_DEVS < <( blkid | awk -F: '/TYPE="swap"/{ print $1 }' )
+   for SWAP in "${SWAP_DEVS[@]}"
+   do
+       if [[ $( grep -q "$( readlink -f "${SWAP}" )" /proc/swaps )$? -eq 0 ]]
+       then
+          err_exit "${SWAP} is already a mounted swap-dev. Skipping" NONE
+          break
```

Comment:
I kinda always forget, but just wanted to check if `break` or `continue` would be the appropriate syntax to "skip" this one while still processing the rest?